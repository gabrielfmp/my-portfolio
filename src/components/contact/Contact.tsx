import {
  ContactWrapper,
  ContactsText,
  IconWrapper,
  StyledIcon,
} from './styles';
import React from 'react';

type ContactProps = {
  icon: string;
  contact: string;
  iconText: string;
};

const Contact: React.FC<ContactProps> = ({ iconText, icon, contact }) => {
  return (
    <ContactWrapper>
      <IconWrapper>
        <StyledIcon alt={iconText} src={icon} />
      </IconWrapper>

      <ContactsText>{contact}</ContactsText>
    </ContactWrapper>
  );
};

export default Contact;
