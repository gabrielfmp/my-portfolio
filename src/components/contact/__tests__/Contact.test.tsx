import { render } from '@testing-library/react';
import Contact from '../Contact';
import Icon from '../../../ui/iconography/phone.svg';
import React from 'react';

test('snapshot', () => {
  const { asFragment } = render(
    <Contact iconText="MyContactIcon" icon={Icon} contact="Contact" />
  );

  expect(asFragment()).toMatchSnapshot();
});
