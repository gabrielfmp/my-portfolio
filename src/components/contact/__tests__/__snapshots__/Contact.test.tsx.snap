// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-gqh2np"
  >
    <div
      class="css-47j6j9"
    >
      <img
        alt="MyContactIcon"
        class="css-12k99am"
        src="phone.svg"
      />
    </div>
    <div
      class="css-un0t8r"
    >
      Contact
    </div>
  </div>
</DocumentFragment>
`;
